### ISSUE TITLE
Suggestion: Wiki

### ISSUE THREAD

**Issue by Fiznik** [Pepe20129/difficultytweaker#1]
Have you considered creating a wiki/documentation explaining what all the configurations do? I was able to figure some out myself but I think it would be really helpful for new users.

Sorry to post this under issues, the mod is super great. Had like over 300 phantoms spawning when players where afk so being able to put a cap on those was sooo good.

**Comment by Pepe20129** [comment]
Yeah I'll definitely be making a wiki, but my priority right now is getting the preference saving system done. Right after I make that I'll make a wiki.
The only problem is that I have to use Cardinal Components API to do that and I'm still figuring out how it works.
Also the command should be able to tell you a lot of info, as an example you can do `/difficultyTweaker bee length` and it'll give you an explanation, the default value and the current value.
